client/src/app/site/site.component.scss-theme.scss

```diff
@@ -37,10 +37,10 @@
         /** style the active link */
         .active {
             mat-icon {
-                color: $os-primary;
+                color: mat-color($primary);
```

Comment:
@emanuelschuetze I would prefer using mat-color here. However, you recently introduced `$os-primary;`